tests/lib/rules/jsx-pascal-case.js

```diff
@@ -29,10 +29,6 @@ const parserOptions = {
 const ruleTester = new RuleTester({parserOptions});
 ruleTester.run('jsx-pascal-case', rule, {
   valid: [{
-    code: '<testComponent />'
-  }, {
-    code: '<test_component />'
-  }, {
```

Comment:
I don't know why these have been considered as valid in the first place. They are not in pascal case.

Is the reasoning that the problem is that they start with a lower case, and thus are wrong to begin with? I can see why we wouldn't want to let `jsx-pascal-case` trigger an error in that situation.

Reply:
This is exactly why: https://github.com/yannickcr/eslint-plugin-react/commit/1e1725997dadf48f8b9131533ed3c1dee2764f54

Reply:
Yes, lowercase components are HTML elements, not custom ones.